src/inc/switches.h

```diff
@@ -233,5 +233,8 @@
 #define FEATURE_STACK_SAMPLING
 #endif // defined (ALLOW_SXS_JIT)
 
+#if defined(USE_INTERPRETER)
+#define FEATURE_INTERPRETER
+#endif // USE_INTERPRETER
```

Comment:
I wonder if, instead of adding another name (USE_INTERPRETER), it would be better to add, here:
```
#if defined(CROSSGEN_COMPILE)
#if defined(FEATURE_INTERPRETER)
#undef FEATURE_INTERPRETER // No interpreter in crossgen builds
#endif
#endif
```
?